core/store/src/trie/mod.rs

```diff
@@ -256,79 +188,6 @@ impl UpdatedTrieStorageNodeWithSize {
     }
 }
 
-impl TrieNode {
-    pub fn has_value(&self) -> bool {
-        match self {
-            Self::Branch(_, Some(_)) | Self::Leaf(_, _) => true,
-            _ => false,
-        }
-    }
-
-    fn memory_usage_for_value_length(value_length: u64) -> u64 {
-        value_length * TRIE_COSTS.byte_of_value + TRIE_COSTS.node_cost
-    }
-
-    fn memory_usage_value(value: &ValueHandle) -> u64 {
-        let value_length = match value {
-            ValueHandle::InMemory(_) => {
-                panic!("InMemory nodes exist, but storage is not provided")
-            }
-            ValueHandle::HashAndSize(value) => u64::from(value.length),
-        };
-        Self::memory_usage_for_value_length(value_length)
-    }
-
-    /// TODO(#12361): in particular, consider replacing with
-    /// `GenericUpdatedTrieNode::memory_usage_direct`.
-    fn memory_usage_direct(&self) -> u64 {
-        match self {
-            TrieNode::Leaf(key, value) => {
-                TRIE_COSTS.node_cost
-                    + (key.len() as u64) * TRIE_COSTS.byte_of_key
-                    + Self::memory_usage_value(value)
-            }
-            TrieNode::Branch(_children, value) => {
-                TRIE_COSTS.node_cost
-                    + value.as_ref().map_or(0, |value| Self::memory_usage_value(value))
-            }
-            TrieNode::Extension(key, _child) => {
-                TRIE_COSTS.node_cost + (key.len() as u64) * TRIE_COSTS.byte_of_key
-            }
-        }
-    }
-}
-
-impl std::fmt::Debug for TrieNode {
```

Comment:
We can give a little thinking if this should be migrated to RawTrieNode. Likely not because we already have some other debug tools

Reply:
We have Debug implemented for RawTrieNode. Would that be good enough?

Reply:
I've meant this "indented" debug where indent depends on node depth but it may be just useless.